ci/ray_ci/requirements.in

```diff
@@ -1,3 +1,4 @@
 click
+colorama
```

Comment:
why is this required now?

colorama historically has introduced issues because of its markdown related dependencies.

Reply:
running the ray_ci tool in windows complaining about this missing dependencies; i can make it a window-specific dependency